src/components/partials/FarmersStatistic.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Statistic, Container, Segment } from 'semantic-ui-react';
+
+const FarmersStatistic = props => {
+  return (
+    <Container style={{ marginBottom: '40px' }}>
+      <Segment.Group horizontal>
+        <Segment textAlign="center" style={{ padding: '14px 0 0 0' }}>
```

Comment:
Here we could use padding and margin to make the stat always centered: `style={{ padding: '14px 0', margin: 'auto 0' }}`